common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -235,6 +235,10 @@ $sequence--border-color: #C8C8C8;
   body.touch-based-device & ol li a:hover p {
     display: none;
   }
+
+  @media print {
```

Comment:
I don't like this. It removes information from the page, particularly, where in the sequence this page was printed from. This can become very important for sequences that have a very large number of units. Is there a way to just print this information better? `@media print` rules can be used in many varied and dynamic ways, not just to show or hide things.